content/logs/index.md

```diff
@@ -207,6 +207,42 @@ logs:
         pattern: (?:4[0-9]{12}(?:[0-9]{3})?|[25][1-7][0-9]{14}|6(?:011|5[0-9][0-9])[0-9]{12}|3[47][0-9]{13}|3(?:0[0-5]|[68][0-9])[0-9]{11}|(?:2131|1800|35\d{3})\d{11})
 ```
 
+### Multi-line
+
+If your logs are not sent in JSON and you want to aggregate several lines into one single log, you can configure the agent to detect a new log thanks to a specific pattern instead of one log per line. This is accomplished by using the `log_processing_rules` parameter in your configuration file with the **multi_line** `type`.
```

Comment:
If your logs are not sent in JSON and you want to aggregate several lines into one single log, you can configure the agent to detect a new log using a specific regex pattern instead of one log per line. This is accomplished by using the `log_processing_rules` parameter in your configuration file with the **multi_line** `type`.
____
~thanks to~ --> using

Reply:
👍